pkg/validate/security_context_linux.go

```diff
@@ -317,6 +317,57 @@ var _ = framework.KubeDescribe("Security Context", func() {
 			Expect(groups).To(ContainElement("5678"))
 		})
 
+		It("if the container's primary UID belongs to some groups in the image, runtime should add SupplementalGroups to them", func() {
+			By("create pod")
+			podID, podConfig, podLogDir = createPodSandboxWithLogDirectory(rc)
+
+			By("create container for security context SupplementalGroups")
+			supplementalGroups := []int64{1234, 5678}
```

Comment:
wdyt about only adding one supplimental group, and using the variable here and in the `expectedOutput := fmt.Sprintf..`  call? Then we could reuse the variable while having the output generating code be relatively simple

Reply:
Thanks for the comment👍 
fixed in 9152103

Reply:
@haircommander PTAL. If the PR looks good to you, please give `/lgtm` for this PR.